server/api.go

```diff
@@ -41,6 +41,7 @@ type SessionResult struct {
 	Disclosed   [][]*irma.DisclosedAttribute `json:"disclosed,omitempty"`
 	Signature   *irma.SignedMessage          `json:"signature,omitempty"`
 	Err         *irma.RemoteError            `json:"error,omitempty"`
+	NextSession irma.RequestorToken          `json:"nextSession,omitempty"`
```

Comment:
Wouldn't you include the whole `SessionPackage` instead of only the requestor token? I can image that for some cases it can also be handy to know the `sessionPtr` too.

Reply:
As discussed in Slack, only the `RequestorToken` should work.